### ISSUE TITLE
Expose operations from TaskChampion

### ISSUE THREAD

**Issue by djmitche** [GothenburgBitFactory/taskwarrior#2928]
We'll need this for a few reasons:
 * To generate a diff for undo (get the list of operations since the last undo point, apply them in reverse, and generate the diff)
 * To generate the "history" of a task (which means we'll need to store operations per-task, until a task is deleted)

**Comment by djmitche** [comment]
@masaeedu is considering working on this, so I'll un-assign from myself.

**Comment by djmitche** [comment]
@ryneeverett I think this is done -- is there anything more to do?

**Comment by ryneeverett** [comment]
Yes, #3213 only addressed the "Operations for Undo" section of your detailed breakdown. "Operations for Info" has not yet been addressed.

**Comment by djmitche** [comment]
Right, thank you!
